### ISSUE TITLE
Questions about using the cross attention model

### ISSUE THREAD

**Issue by urnotcoward** [Messi-Q/AMEVulDetector#3]
Dear professor Peng Qian，
Recently I have read the latest paper published by your team in IJCAI-21-《Smart Contract Vulnerability Detection: From Pure Neural Network to Interpretable Graph Feature and Expert Pattern Fusion》, which is a very interesting work and has brought me a lot of inspiration.
      I have read your paper and code, and want to use the cross attention mechanism mentioned in your paper to perform feature fusion on the data.
![image](https://user-images.githubusercontent.com/71996208/173348369-deaee375-cbd9-4883-88f6-56c7dbf417b5.png)

       
       The following is the model I adjusted according to the idea of cross attention to perform feature fusion, but the effect is very poor. Before the fusion, the accuracy of my data classification was 90%, but after using the cross attention feature fusion model, the accuracy dropped to less than 50%.

       I tried to ask related questions in the TensorFlow community, but none of the community members seem to understand this new model and mechanism.
       Would you please to help me find out what is wrong with my model？
       I would like to hear your experience and advice about it.

![image](https://user-images.githubusercontent.com/71996208/173348409-30d58696-4d95-4f3b-8ca0-e91caf6e59fd.png)

        I'm very sorry if it bothered you. Looking forward to your reply！
        All the best to you and your team~☺

**Comment by scener-y** [comment]
你好，打扰了，我还没有做的这一步，我对标签的问题一直疑惑，不知道怎么获取数据集的标签，作者在文件里面给的数据的标签很少，请问如何获取更多的智能合约漏洞标签。谢谢。

**Comment by scener-y** [comment]
Hello, sorry to bother you. I haven't taken this step yet. I have been puzzled about the issue of tags and don't know how to obtain the tags of the dataset. The author provided very few tags for the data in the file. How can I obtain more smart contract vulnerability tags. Thank you.
